freqtrade/freqtradebot.py

```diff
@@ -694,8 +709,10 @@ def check_handle_timedout(self) -> None:
             if order['status'] == 'open':
                 if order['side'] == 'buy' and ordertime < buy_timeoutthreashold:
                     self.handle_timedout_limit_buy(trade, order)
+                    self.wallets.update()
```

Comment:
i tend to think that you'll need that call a few lines higher too - this is only the "handle timeout" part - if it completed successfully (remaining ==0) - then it never reaches this point ...